src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/interlocsendtotcw/InterlocSendToTcwAboutToSubmitHandler.java

```diff
@@ -33,9 +34,9 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
 
         SscsCaseData caseData = callback.getCaseDetails().getCaseData();
 
-        log.info("Setting InterlocReviewState to null for case id {} "
-                + "for AppealDormantAboutToSubmitHandler" + callback.getCaseDetails().getId());
-        caseData.setInterlocReviewState(null);
```

Comment:
interlocReviewState is removed, is it expected?

Reply:
Yes that is expected